hydroshare/local_settings.cutover

```diff
@@ -198,3 +198,9 @@ HSWS_GEOSERVER_URL = "https://geoserver.hydroshare.org/geoserver"
 HSWS_GEOSERVER_ESCAPE = {
     '/': ' '
 }
+
+DEFAULT_AUTHENTICATION_CLASSES = (
```

Comment:
I added this block to the local_settings templates because the OIDCBasicAuthentication is not used for local development.

Reply:
I actually don't know when/where local_settings.cutover is used. [We use local_settings.template for local dev](https://github.com/hydroshare/hydroshare/blob/2.13/local-dev-first-start-only.sh#L271)

Reply:
I have been confused all along what the cutover vs template relationship is.

Reply:
think I discovered that our PR builds use the `.cutover` and the `.template` is used in other builds